src/lib/api.js

```diff
@@ -0,0 +1,113 @@
+import fetch from 'isomorphic-unfetch';
+
+const API_URL = process.env.API_URL;
+
+async function fetchData(url) {
+  return (await fetch(url)).json();
+}
+
+export function getHome({ fiscalYear }) {
+  let url = `${API_URL}/pages/home`;
+
+  if (fiscalYear) {
+    url = `${url}?fiscal_year=${fiscalYear}`;
+  }
+
+  return fetchData(url);
+}
+
+export function searchContracts({ query, page, pageSize }) {
+  let url = `${API_URL}/contracts/?search=${query}&page=${page}&page_size=${pageSize}`;
```

Comment:
Use a query string lib for generate this string safely? https://www.npmjs.com/package/query-string